docs/pages/Configuration.md

```diff
@@ -702,6 +705,24 @@ Version | UI5 CLI Release
 **1.0** | v1.0.0+
 **0.1** | v0.0.1+
 
+### Specification Version 3.0 [beta]
+
+!!! info
+    **Note:** UI5 Tooling version 3.0 is currently in development. If you wish to migrate to the latest UI5 Tooling, please check the [Upgrade Guide for v3](../../updates/migrate-v3/)
```

Comment:
**Note:** UI5 Tooling version 3.0 is currently in development. If you wish to migrate to the latest UI5 Tooling, check the [Upgrade Guide for v3](../../updates/migrate-v3/)

Reply:
Fixed in https://github.com/SAP/ui5-tooling/pull/735